app/models/grda_warehouse/tasks/push_clients_to_cas.rb

```diff
@@ -52,7 +52,15 @@ def sync!
               source_clients: { most_recent_pathways_or_rrh_assessment: [:assessment_questions, :user] },
               cohort_clients: :cohort,
               source_enrollments: [:income_benefits, :exit],
-            ).
+            ]
+            if RailsDrivers.loaded.include?(:eccovia_data) && EccoviaData::Fetch.exists?
```

Comment:
For the future, this seems like the kind of thing that would be well served by the plug-in pattern for drivers communicating back to the core like we do with sub-populations.